android/CouchbaseLite/src/main/java/com/couchbase/lite/DatabaseConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package com.couchbase.lite;
+
+import android.content.Context;
+
+public class DatabaseConfiguration extends BaseDatabaseConfiguration{
+    private Context context;
+
+    public DatabaseConfiguration() {
```

Comment:
I think we just don't allow. Otherwise we will need to check whether users set directory or not and throw runtime exception. In the future, if this becomes a concern of difficult to use, we can add back. What do you think?